recipe/meta.yaml

```diff
@@ -41,14 +41,27 @@ test:
     - pandas
   commands:
     - pip check
+    # Skip test suite on PyPy as it segfaults there
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/148
+    - python -c "import pandas; pandas.test()"  # [python_impl == "cpython"]
   requires:
     - pip
+    - pytest >=6.0
+    - pytest-asyncio >=0.17
+    - pytest-xdist >=1.31
+    - pytest-cov
+    - boto3
+    - hypothesis
+    - psutil
 
 about:
   home: http://pandas.pydata.org
   license: BSD-3-Clause
   license_file: LICENSE
-  summary: High-performance, easy-to-use data structures and data analysis tools.
+  summary: >-
+    Powerful data structures for data analysis, time series, and statistics
+  doc_url: https://pandas.pydata.org/pandas-docs/stable
+  dev_url: https://github.com/pandas-dev/pandas
```

Comment:
Went ahead and updated the metadata here [inline with what is now recommended]( https://github.com/conda-forge/staged-recipes/blob/5c4388e092bb199dbc9760b4c2e15bf59ed16659/recipes/example/meta.yaml#L66 ). Filled out the values based on [the upstream metadata]( https://github.com/pandas-dev/pandas/blob/v1.5.2/setup.cfg ).